### PR TITLE
Wins refactor js

### LINKED ISSUE #1818: Refactor Wins page JS code
### Overview
The script for the wins page could use a little bit of clean up. In the interest of maintainability and readability, the code may need a bit of revision.

Before starting, please see [this comment](https://github.com/hackforla/website/issues/1818#issuecomment-879535414).

### Action Items
- [x] Obtain an understanding of the JS code for the wins page and the way that code functions
  - [x] Take care to understand where inputs are coming from and outputs are going.
- [x] Add comments to lines that might be confusing, or require explanation
- [x] Conversely, remove comments that might not be required
- [x] Rename variables if the names are not intuitive
- [x] Condense code where doing so would improve clarity and conciseness
- [x] Do not break anything! The page should function the same way before and after
- [x] That said, if any bugs are noticed, quietly create a new issue for them (or tack it on here as an action item if you decide to tackle them with this issue). Templates are provided below.
  - [ ] Bug 1: [add bug here]
  - [ ] Bug 2: [add bug here]

### Resources/Instructions
This is a very freeform issue. It can be as big or as small as you want to be. The goal here is for future developers to discover this code, and be able to quickly understand the code and make changes. A large part of this is understanding our codebase and we encourage you to ask questions!

Some resources:
[Commenting](https://www.cs.utah.edu/~germain/PPS/Topics/commenting.html)
[Writing Comments in Python](https://realpython.com/python-comments-guide/)
[Mozilla's JavaScript Guide](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide)
[wins directory](https://github.com/hackforla/website/tree/wins-feature-1/pages/wins)

Note: Only wins-share-form.html and wins.html should have JS code.

### PR DESCRIPTION
**Pull request by Abel-Zambrano**
Fixes #1818

### What changes did you make and why did you make them ?

- Clarified comment on line 107
- Change function to arrow function
- removed extra spaces

### CODE REVIEW

File: pages/wins/wins.html [hunk 1]
```diff
@@ -89,7 +89,7 @@
   	//cardData[index][question_str] = answer_str
   	let cardData = []
   	const keys = rawData[0]
-  	rawData.slice(1).forEach((data) => {
+  	rawData.slice(1).forEach( data => {
```
Comment (adam-abundis) [inline pages/wins/wins.html]: please remove the space between `(` and `data` so they are next to each other

File: pages/wins/wins.html [hunk 2]
```diff
@@ -169,7 +167,7 @@
   	let queryObj = { };
 
   	//Calculate and Create Updated Query String
-  	incomingFilterData.forEach(function(e){
+  	incomingFilterData.forEach( e => {
```
Comment (adam-abundis) [inline pages/wins/wins.html]: Please remove the space between `(` and `e`  so they are next to each other

File: pages/wins/wins.html [hunk 3]
```diff
@@ -296,20 +290,20 @@
 
 		const iconContainer = cloneCardTemplate.querySelector(cardSelector);
 
-		cardList.forEach( (item) => {
+		cardList.forEach( item => {
```
Comment (adam-abundis) [inline pages/wins/wins.html]: Please remove the space between `(` and `item`  so they are next to each other

**Review by adam-abundis — CHANGES_REQUESTED** [review]
Good job @Abel-Zambrano. The changes look good. I only have a few formatting requests. To keep consistent with the rest of the code, can you please remove the space in the arrow functions so the argument is right next to the opening parenthesis. I have noted which lines to change. Thank you.

**Comment by Abel-Zambrano** [comment]
@abuna1985 The changes are done.

**Review by Zak234 — APPROVED** [review]
Everything looks good!

**Review by adam-abundis — APPROVED** [review]
@Abel-Zambrano Thank you for making the changes. Everything looks good to me.

**Pull request merged**